docs/templates/manual.html.j2

```diff
@@ -33,7 +33,17 @@
       {%- endmacro %}
       <main id="main" class="col-md-9" data-bs-spy="scroll" data-bs-target="#contents" data-bs-threshold="0,1" data-bs-root-margin="-30% 0% -70%">
         <h1>{{ headline }}</h1>
-        {{ history | markdownify }}
+        <p>
+          <em>
+            For other versions, see
+            <a href="/jq/manual/v1.7/">1.7</a>,
+            <a href="/jq/manual/v1.6/">1.6</a>,
+            <a href="/jq/manual/v1.5/">1.5</a>,
+            <a href="/jq/manual/v1.4/">1.4</a>,
+            <a href="/jq/manual/v1.3/">1.3</a> or
+            <a href="/jq/manual/dev/">development version</a>.
```

Comment:
Made these same and absolute for all for less maintenance work and so that links are correct at both /jq/manual/ and /jq/manual/<version>/

Reply:
The `/jq` looks like the `--root` and the links don't work locally?

Reply:
Wops i messed up the markdown. It was suppose to say "...are correct at both `/jq/manual/` and `/jq/manual/<version>/`". If we want to use a symlink to point /jq/manual/ to current release then that manual will be accessable at two locations so `../` will not work.

With the caddy config that maps `/jq/<path>` to  `<docs output root>/<path>` (same as https://jqlang.github.io/jq/) links work fine for me locally. Or did you mean something else?

I think we want to keep `/jq/manual/` working and not break links?

Reply:
As mentioned in [docs/README.md](https://github.com/jqlang/jq/tree/master/docs) and  [manual](https://jqlang.github.io/jq/download/), we can specify the root path with `--root` to make links working with the `output/` dir, serving with Python easily. So I think the links don't work locally. But the `--root` is a mess, I forgot why I couldn't make links work without this option, so you can cleanup the option.

Reply:
Ah i see, didn't know. I fixed the link to be relative to root now so i think we can keep it for now.